FreeRTOS/Test/CMock/smp/single_priority_timeslice/single_priority_timeslice_utest.c

```diff
@@ -76,6 +76,29 @@ int suiteTearDown( int numFailures )
     return numFailures;
 }
 
+/* ==============================  Helper function  ============================== */
```

Comment:
Can you move this function to smp_utest_common.c/h?
The remaining set of test cases will also require this function